blosc/blosc2.h

```diff
@@ -900,6 +900,21 @@ BLOSC_EXPORT int blosc2_getitem_ctx(blosc2_context* context, const void* src,
 #define BLOSC2_MAX_METALAYERS 16
 #define BLOSC2_METALAYER_NAME_MAXLEN 31
 
+/**
+ * @brief This struct is meant for holding storage parameters for a
+ * for a blosc2 container, allowing to specify, for example, how to interpret
+ * the contents included in the schunk.
+ */
+typedef struct {
+    bool sparse;  //!< Whether the chunks are sparse or sequential (frame)
+    char* path;   //!< The path for storage; if NULL, that means in-memory
```

Comment:
Add a 'else' description?

Reply:
I don't think is necessary, as the 'else' is described at the beginning.  But I added a bit more detailed description in 8db4718.